extension.json

```diff
@@ -0,0 +1,53 @@
+{
+	"@Note": "When updating this file please also update SemanticFormsSelect.php with the same changes.",
+	"load_composer_autoloader": true,
+	"name": "Semantic Forms Select",
+	"version": "1.3.0",
+	"author": [
+		"Jason Zhang",
+		"Toni Hermoso Pulido",
+		"..."
+	],
+	"url": "https://www.mediawiki.org/wiki/Extension:SemanticFormsSelect",
+	"descriptionmsg": "semanticformsselect-desc",
+	"type": "semantic",
+	"requires": {
+		"MediaWiki": ">= 1.27.0"
+	},
+	"license-name": "GPL-2.0+",
+	"callback": "SFS\\SemanticFormsSelectHooks::onCallback",
```

Comment:
I think referencing PHP constructs like this is fallacy. The other stuff is cool and all, but with those refs I'm not going to +1 this change.